tools/tinytest-codegen.py

```diff
@@ -127,8 +141,10 @@ def load_profile(profile_file, test_dirs, exclude_tests):
         test_dirs, exclude_tests = load_profile(args.profile, test_dirs, exclude_tests)
     if args.exclude:
         exclude_tests = exclude_tests.union(args.exclude)
-    for group in test_dirs:
-        tests += [test for test in glob("{}/*.py".format(group)) if test not in exclude_tests]
+    for group in sorted(test_dirs):
+        tests += [
+            test for test in sorted(glob("{}/*.py".format(group))) if test not in exclude_tests
+        ]
```

Comment:
Wouldn't it be a lot simpler to just do `tests.sort()` after this for loop?  Then there's no need to flatten anything.

Reply:
@agatti any thought about using `sort()` to keep this simple?

Reply:
Apologies for the delay, this fell off my radar.  The reason for the list flattening was to prevent potential user error, as it was not obvious the tests exclusion list is a set and not an array of sorts.

If that's not a concern I can sort the final list without pre-flattening the input as you suggest.